IMC_WebDev/Introduction_to_CSS/proposal.md

```diff
@@ -0,0 +1,22 @@
+TITLE:
+Introduction To CSS
```

Comment:
This title seems a bit too general and matter-of-fact. Blog titles should be informative but appealing, and make you wat to click on it.